sdmetrics/visualization.py

```diff
@@ -88,15 +88,15 @@ def _generate_heatmap_plot(all_data, columns):
     """
     unique_values = all_data['Data'].unique()
 
+    if len(columns) != 2:
+        raise ValueError('Generating a heatmap plot requires only two columns for the axis.')
```

Comment:
I got confused with this and the other PR about getting a column plot 😅 . I don't think you need to assert that exactly 2 columns are passed and you can get rid of those tests. If you want to leave it can you change the following
```suggestion
    if len(columns) != 2:
        raise ValueError('Generating a heatmap plot requires exactly two columns for the axis.')
```